salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -132,8 +119,17 @@ server {
         {% endif %}
     }
 
+    location = /favicon.ico {
+        proxy_pass http://localhost:8080/favicon.ico;
+    }
+
     location /assets {
         add_header Cache-Control "public, max-age={{ 60 * 60 * 24 * 365 }}, immutable";
```

Comment:
Should probably be in the `assets` Nginx (and combine the `location`s)?

Reply:
In fact, do we even need a container for these? Could copy the first to the host instead?

Reply:
Forward looking, containers are all there is as in an orchestrator there is no accessible host; trying to minimize what the host nginx does as eventually it should disappear.

Reply:
Fair enough. 👍